pkg/pluginversion/pluginversion.go

```diff
@@ -37,13 +37,21 @@ func CheckPluginExists(project string, version string) (bool, error) {
 	// FIXME: would be nice if we could use `HEAD` here
 	resp, err := http.Get(pluginUrl)
 
-	if resp.StatusCode == http.StatusOK {
-		return true, nil
+	// We got an error, bail
+	if err != nil {
+		return false, err
 	}
+
+	// Not found, there's no plugin yet
 	if resp.StatusCode == http.StatusNotFound {
```

Comment:
Do we need to check for an empty response? Is there anyway that this can happen?

Reply:
Yes we probably do

Reply:
actually, on second thoughts, I think no statuscode == error, which we're returning, so i think this is okay.